static/app/components/performance/spanSearchQueryBuilder.tsx

```diff
@@ -54,8 +107,8 @@ export function SpanSearchQueryBuilder({
   });
 
   const filterTags: TagCollection = useMemo(() => {
-    return {...supportedTags};
-  }, [supportedTags]);
+    return Object.assign({}, functionTags, supportedTags);
```

Comment:
nit: it's the same thing but generally we just do `{...functionTags, ...supportedTags}`